src/ext.js

```diff
@@ -469,6 +469,20 @@ const getDefinition = (env) => {
             translation: 'properties.presentation',
             items: getStyleSettings(env),
           },
+          ...(env.flags.isEnabled('PS_18291_TABLE_EXPLORATION') && {
```

Comment:
what happens when `isEnabled` returns false, does it still work to use `...`? Also saw that the a test is failing. You need to mock `flags`

Reply:
If flag is set `...` will spread the second part after && if flag is not set `...` will spread `{}`

Reply:
ok nice :)

Reply:
Why spread it though?

Reply:
@MTjody The second part should only be added to the definition if the flag is set. the spread is to attach the second part in to the parent object.

Reply:
It does something like this
```
const person = { name: "John Doe",   ...( showOccupation && {occupation: "engineer"})};
setShowOccupation(false);
// person = { name: "John Doe"}
setShowOccupation(true);
// person =  { name: "John Doe", occupation: "Engineer"};
```